Hunting Queries/Corelight/CorelightRepetitiveDnsFailures.yaml

```diff
@@ -0,0 +1,29 @@
+id: 1ea3a384-77a4-4b0d-8e42-74d26b38ab5b
+name: Corelight - Repetitive DNS Failures
+description: |
+  'Query searches for repetitive DNS resolution failures from single host.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated